pkg/db/db.go

```
@@ -64,32 +71,52 @@ func createTable(ctx context.Context, pool *pgxpool.Pool, tableName string) erro
+			time_bucket('1 day', time) AS day,
+			AVG(low) as average
+		FROM
+			%s
+		GROUP BY
+			day
+    `, dailyAverageView, ohclTable))
+
+	// TODO: if successful "CREATE MATERIALIZED VIEW"
+	fmt.Printf("result: %v\n", result.String())
+
+	_, err = pool.Exec(ctx, fmt.Sprintf(`
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)